icons/git-merge-conflict.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "../icon.schema.json",
+    "contributors": [
+        "julczka",
```

Comment:
She only requested the icon, but didn't create it.

Since this is heavily based on https://lucide.dev/icons/git-pull-request and https://lucide.dev/icons/git-pull-request-closed, it is probably a good idea to list each contributor of those icons as well.